esmvaltool/recipes/recipe_autoassess_landsurface_surfrad.yml

```diff
@@ -0,0 +1,75 @@
+# ESMValTool
+# recipe_autoassess_landsurface_surfrad.yml
+---
+documentation:
+  description: |
+    Recipe that runs the Autoassess Land-surface assessment area diagnostic.
+    Land-sea masks are not averaged over time (annual) but are fixed, i.e.
+    using sftlf (lat-lon) grid. The original  Autoassess diagnostic used
+    time-dependent .pp masks these can still be implemented if present in
+    autoassess_source. The recipe is fully working but masks need to be
+    checked.
+
+  authors:
+    - predoi_valeriu
+
+  maintainer:
+    - predoi_valeriu
+
+datasets:
+  - {dataset: MPI-ESM-LR,  project: CMIP5,  exp: amip,  ensemble: r1i1p1,  start_year: 1992,  end_year: 2002}
+  - {dataset: MPI-ESM-MR,  project: CMIP5,  exp: amip,  ensemble: r1i1p1,  start_year: 1992,  end_year: 2002}
+  - {dataset: inmcm4,      project: CMIP5,  exp: amip,  ensemble: r1i1p1,  start_year: 1992,  end_year: 2002}
+
+preprocessors:
+  pp_aa_area:
+    regrid:  # NOT USED
+      target_grid: 0.15x0.15
+      scheme: linear
+
+diagnostics:
+  aa_landsurf_surfrad:
+    description: Autoassess test diag for Land-Surface Surfrad.
+    variables:
+      rsns: # Surf SW net all sky
```

Comment:
I think this should be rss instead of rsns. In fact I can't find rsns in the CMIP6 variable names - is this a name that changed from CMIP5 to CMIP6?

Reply:
`rsns` is a derived variable with its own [custom table(https://github.com/ESMValGroup/ESMValCore/blob/master/esmvalcore/cmor/tables/custom/CMOR_rsns.dat) and [derivation script](https://github.com/ESMValGroup/ESMValCore/blob/master/esmvalcore/preprocessor/_derive/rsns.py) - `rss` is this guy:
```
        "rss": {
            "frequency": "mon",
            "modeling_realm": "atmos",
            "standard_name": "surface_net_downward_shortwave_flux",
            "units": "W m-2",
            "cell_methods": "area: time: mean",
            "cell_measures": "area: areacella",
            "long_name": "Net Shortwave Surface Radiation",
            "comment": "Net downward shortwave radiation at the surface",
            "dimensions": "longitude latitude time",
            "out_name": "rss",
            "type": "real",
            "positive": "down",
            "valid_min": "",
            "valid_max": "",
            "ok_min_mean_abs": "",
            "ok_max_mean_abs": ""
        },
        
```
and is Emon instead of Amon

Reply:
Ah, OK - the derived variable is fine. It's exactly the same as rss. I'm guessing that rss didn't exist at CMIP5 and so the derived variable was needed.

Reply:
cool! cheers Ali :beer: